pkg/remote/conncontroller/conncontroller.go

```
@@ -377,6 +377,28 @@ to ensure a seamless experience.
 Would you like to install them?
 `)
 
+func (conn *SSHConn) UpdateWsh(ctx context.Context, clientDisplayName string, opts *WshInstallOpts) error {
+	if opts == nil {
+		opts = &WshInstallOpts{}
```

golangci-lint (1.62.2)

ineffectual assignment to opts

(ineffassign)